lib/atomic_lti_1v1/lti_1v1.rb

```diff
@@ -1,7 +1,10 @@
 module AtomicLti1v1
   class Lti1v1
     def self.is_lti_1v1?(request)
-      request.params['oauth_consumer_key'].present?
+      contains_oauth_consumer_key = request.params['oauth_consumer_key'].present? 
+      path_matches = request.env["PATH_INFO"].match(AtomicLti1v1.lti_launch_path_regex)&.size == 1
```

Comment:
I wonder if we should allow an array of launch paths. There could be multiple

Reply:
Good call

Reply:
Though this is a regex so I guess you could already do that. Maybe we should say it shouldn't be a regex and instead be a list of paths?

Reply:
Do list of paths that are interpreted as path prefixes